extensions/applicationinsights-dependencies-js/src/ajax.ts

```diff
@@ -456,30 +474,53 @@ export class AjaxMonitor extends BaseTelemetryPlugin implements IDependenciesPlu
                 }
             }
 
+            function _isDisabledRequest(xhr?: XMLHttpRequestInstrumented, request?: Request | string, init?: RequestInit) {
+                let isDisabled = false;
+                let theUrl:string = (!CoreUtils.isString(request) ? (request as Request).url || "" : request as string).toLowerCase();
```

Comment:
I actually pass the url as part of the call from the XHR, so no.

However, if they don't specify a url or pass undefined this would crash -- I'll add an extra check (if I don't have one already)